css/fieldmanager.css

```diff
@@ -30,6 +42,37 @@
 	margin: 0 0 0 5px;
 }
 
+.fm-label label {
+	font-size: 13px;
+	font-weight: 700;
+	line-height: 1.4em;
```

Comment:
In general I'd recommend unitless values for `line-height`, as they will work regardless of the unit you use for `font-size`.